src/components/Popups/ReplyField/ReplyField.tsx

```diff
@@ -125,23 +174,36 @@ export default class ReplyField extends React.Component<Props, State> {
     };
 
     render(): JSX.Element {
-        const { className, isDisabled, placeholder, ...rest } = this.props;
+        const { className, isDisabled, itemRow, placeholder, ...rest } = this.props;
         const { editorState, popupReference } = this.state;
 
         return (
             <div className={classnames(className, 'ba-ReplyField')}>
-                <Editor
-                    {...rest}
-                    ref={this.editorRef}
-                    editorState={editorState}
-                    onChange={this.handleChange}
-                    placeholder={placeholder}
-                    readOnly={isDisabled}
-                    stripPastedStyles
-                    webDriverTestID="ba-ReplyField-editor"
-                />
-
-                {popupReference && <PopupList reference={popupReference} />}
+                <PopupList
```

Comment:
It seems a little odd for PopupList to be a parent to Editor. Can they remain siblings and communicate via callbacks and shared local state?